src/arguments/CoreMessage.ts

```diff
@@ -14,7 +14,7 @@ export class CoreArgument extends Argument<Message> {
 
 	public async run(parameter: string, context: MessageArgumentContext): AsyncArgumentResult<Message> {
 		const channel = context.channel ?? context.message.channel;
-		const message = (await this.resolveByID(parameter, channel)) ?? (await this.resolveByLink(parameter, context));
+		const message = (await this.resolveById(parameter as Snowflake, channel)) ?? (await this.resolveByLink(parameter, context));
```

Comment:
```suggestion
		const message = (await this.resolveById(parameter, channel)) ?? (await this.resolveByLink(parameter, context));
```

And change it to `parameter: Snowflake` on line 15

Reply:
That's not correct because it can also be an URL.